configs/babel.js

```diff
@@ -82,6 +82,7 @@ module.exports = function (api) {
 
 			// Stage 3
 			// '@babel/plugin-syntax-import-meta',
+			[require('@babel/plugin-proposal-class-properties').default, {loose: true}],
```

Comment:
Should we add `@babel/plugin-proposal-class-properties` dependency to package.json?

Reply:
Don't need to.
It's already included in `@babel/preset-env` so we can use without adding in package.json